src/main/resources/application.yaml

```diff
@@ -113,6 +113,8 @@ spring:
             loggerLevel: ${FEIGN_LOG_LEVEL:none}
 
 darts:
+  event-obfuscation:
+    enabled: ${EVENT_OBFUSCATION_ENABLED:false}
```

Comment:
Probably also worth enabling this for local dev in `application-local.yaml`.

Reply:
For local development we use environment properties to override these values.
(This approach aligns to the one we use for manual deletion)

Reply:
Our approach isn't totally consistent but the general vision (for local development) has been to minimise which env vars we need to set (generally limited to secrets), and have everything else set to some sensible default in `application-local.yaml`. The goal is to make getting stood up for local development easier.

I'd think (?) for local dev we'd want this (and probably any other feature flags?) enabled 99.9% of the time, so it would make the dev experience a bit nicer if we can default it to true when running with the local profile (so in `application-local.yaml`):
```
  event-obfuscation:
    enabled: ${EVENT_OBFUSCATION_ENABLED:true}
```

Then we can still use an env var to override it if we really want to? Just one less thing to think about...